offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/dataresources/products/ProductsDbConnector.groovy

```diff
@@ -220,11 +224,87 @@ class ProductsDbConnector {
     return productItems
   }
 
+  /**
+   * A product is archived by setting it inactive
+   * @param product The product that needs to be archived
+   * @since 1.0.0
+   * @throws life.qbic.business.exceptions.DatabaseQueryException
+   */
+  @Override
+  void archive(Product product) throws DatabaseQueryException {
+    Connection connection = provider.connect()
+
```

Comment:
Currently there is no test for archiving a product is this implemented in another PR?

Reply:
How do we want to handle the reversion of an archived product?

Reply:
yes tests are with the business logic